.github/workflows/clean_pr.yml

```
@@ -19,4 +19,14 @@ jobs:
             printf 'The following files were both added and deleted in this PR:\n%s\n' "$ad"
             exit 1
           fi
-
+      - name: Check for added-and-modified images
+        run: |
+          git fetch --quiet origin "$GITHUB_BASE_REF"
+          base="$(git merge-base "origin/$GITHUB_BASE_REF" 'HEAD^2')"
+          am="$(git log "$base..HEAD^2" --pretty=tformat: --name-status --diff-filter=AM |
+                cut --fields 2 | sort | uniq --repeated |
+                grep -E '.(png|pdf|ps|eps|svg)')"
+          if [[ -n "$am" ]]; then
+            printf 'The following images were both added and modified in this PR:\n%s\n' "$ad"
```

```suggestion
            printf 'The following images were both added and modified in this PR:\n%s\n' "$am"
```
Unless this is some bash magic my naive bash understanding is not aware of.

---

re-woops.